spec/features/applications/revoke_approved_application_spec.rb

```diff
@@ -0,0 +1,134 @@
+require 'rails_helper'
```

Comment:
In the future you can use our joins table and explicitly set all of these parts instead of having to manually go through the process of clicking buttons and filling out forms